DEPLOY-310 Update the DBP deploy to use the common components we moved from the Activiti org

jottley opened the pull request:
Move activemq, alfresco-activiti-cloud-registry, and alfresco-dbp-infrastructure charts to this github repo.

Update the README to use a new flow to install the infrastructure services for the DBP

Commit:
DEPLOY-177: Identify components needed for the cloud registry and build up/reuse helm charts
Added initial version for Cloud-Registry Chart

Commit:
DEPLOY-177: Identify components needed for the cloud registry and build up/reuse helm charts
Changed pull policy to Always as per review comment on Deploy-178

Commit:
DEPLOY-177: Identify components needed for the cloud registry and build up/reuse helm charts
Addressed review comment (removed probes)

sergiuv2020 committed:
DEPLOY-166 / DEPLOY-197 : Update Readme and infrastructure chart folder name

sergiuv2020 committed:
DEPLOY-197 / DEPLOY-284 Update dbp chart to containe new acs chart and new images. Temporary ACS chart placed here

sergiuv2020 committed:
DEPLOY-197 / DEPLOY-284 : Fix sync service svc and ingress, modified default values for dbp

sergiuv2020 committed:
DEPLOY-279 : add condition for enabling nginx, otherwise build would push an incorrect chart to the repo

nicdoye committed:
DEPLOY-294 Move ActiveMQ into Infrastructure chart

DEPLOY-294 Merge changes from DEPLOY-311

DEPLOY-294 rename and refactor activemq port vars in YAML

DEPLOY-294 Move ActiveMQ into Infrastructure Chart

nicdoye committed:
DEPLOY-294 ActiveMQ in separate helm chart

nicdoye committed:
DEPLOY-294 ActiveMQ in separate helm chart

nicdoye committed:
DEPLOY-294 Improved init check for ActiveMQ

ffazzini committed:
Required changes to integrate activiti

ffazzini committed:
Required changes to integrate activiti

sergiuv2020 committed:
Added rabbitmq non clustered activation by default, and minor fixes

sergiuv2020 committed:
Added rabbitmq non clustered activation by default, and minor fixes

sergiuv2020 committed:
Added default to eurekaUrl variable, and also remove the ingress for it since it should not be an external facing service. Changed rabbitmq-ha replica number to 1

sergiuv2020 committed:
Added default to eurekaUrl variable, and also remove the ingress for it since it should not be an external facing service. Changed rabbitmq-ha replica number to 1

sergiuv2020 committed:
added condition and enabled value for activemq in infra chart

sergiuv2020 committed:
AUTH-15 : Fix registry pull secrets for activiti charts

sergiuv2020 committed:
AUTH-15 : Merge master in AUTH-15

sergiuv2020 committed:
DEPLOY-352: fixed the version of Nginx-Ingress, removed invalid path for the gateway

sergiuv2020 committed:
DEPLOY-352: fixed the version of Nginx-Ingress, removed invalid path for the gateway

sergiuv2020 committed:
DEPLOY-352 Update ingress version and correct ingress definition for keycloak

sergiuv2020 committed:
DEPLOY-352: Force nginx controller scope to release namespace

jottley committed:
[DEPLOY-310] Update the DBP deploy to use the common components we moved from the Activiti org
Prepare move of activemq chart to alfresco-infrastructure-deployment repo

jottley committed:
Merge activemq chart from alfresco-dbp-deployment repo to alfresco-infrastructure-deployment repo for DEPLOY-310

jottley committed:
[DEPLOY-310] Update the DBP deploy to use the common components we moved from the Activiti org
Prepare move of alfresco-dbp-infrastruacture chart to alfresco-infrastructure-deployment repo

jottley committed:
Merge move of alfresco-dbp-infrastructure chart from alfresco-dbp-deployment repo to alresco-infrastructure-deployment repo for  DEPLOY-310

jottley committed:
[DEPLOY-310] Update the DBP deploy to use the common components we moved from the Activiti org

Point activemq dependency in requirements.yaml to the charts incubator

jottley committed:
[DEPLOY-310] Update the DBP deploy to use the common components we moved from the Activiti org

Add alfresco-keycloak, alfresco-api-gateway, and alfresco-activiti-cloud-registry and dependencies to the alfresco-dbp-infrastrucutre chart

jottley committed:
[DEPLOY-310] Update the DBP deploy to use the common components we moved from the Activiti org

Prepare move of alfresco-activiti-cloud-registry from alfresco-dbp-deployment repo to alfresco-infrastructure-deployment repo

jottley committed:
Merge move of alfresco-activiti-cloud-registry from alfresco-dbp-deployment repo to alfresco-infrastructure-deployment for DEPLOY-310

jottley committed:
Merge branch 'DEPLOY-310' of https://github.com/Alfresco/alfresco-infrastructure-deployment into DEPLOY-310

jottley committed:
Merge branch 'DEPLOY-310' of https://github.com/Alfresco/alfresco-infrastructure-deployment into DEPLOY-310

jottley committed:
[DEPLOY-310] Update the DBP deploy to use the common components we moved from the Activiti org

Remove ngnix-ingress dependency. The ingress now needs to be installed prior to the infrastructure

jottley committed:
[DEPLOY-310] Update the DBP deploy to use the common components we moved from the Activiti org

Remove ngnix-ingress dependency.  It needs to be installed before the the infrastructure is installed

jottley committed:
Add .gitignore

jottley committed:
[DEPLOY-310]  Update the DBP deploy to use the common components we moved from the Activiti org

Update README for installing the infrastructure

jottley committed:
[DEPLOY-310] Update the DBP deploy to use the common components we moved from the Activiti org

Remove setting of rabbtmq release name in the README.  It is not currently needed.

rgauss commented on README.md:
```diff
@@ -1 +1,166 @@
-# alfresco-infrastructure-deployment
\ No newline at end of file
+# Alfresco Digital Business Platform Infrastructure Deployment
```
We shouldn't refer to DBP in this repo anymore due to branding concerns.

jottley replied:
Ok. Any suggestions on the name?

rgauss replied:
Just 'Alfresco Infrastructure Deployment' I think.

rgauss commented on README.md:
```diff
@@ -1 +1,166 @@
+
+This environment variable will be used in the deployment steps.
+
+### Docker Registry Pull Secrets
```
We didn't define whether or not pushing the relevant Docker images to Docker Hub was part of DEPLOY-310.

If it is then this isn't needed.

jottley replied:
Oh yeah...all of these images should be in public locations...I'll remove.

rgauss commented on charts/incubator/alfresco-dbp-infrastructure/Chart.yaml:
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart Providing the required Infrastructure deployments for the Alfresco DBP. This includes nginx-ingress, activemq, alfresco-api-gateway, alfresco-keycloak, and the content services volume.
+name: alfresco-dbp-infrastructure
```
This should just be named `alfresco-infrastructure`.  We should not have any references to DBP in this GitHub repo.

jottley replied:
Ok.  There will be a couple of places that I need to update this. Will make the changes.

jottley committed:
[DEPLOY-310] Update the DBP deploy to use the common components we moved from the Activiti org

Remove references to DBP or Digital Business Platform.
Remove section on setting up the Docker Registry Pull Secrets.  Infrastrucutre images are all publically available.

sergiuv2020 reviewed (approved):
LGTM

jottley committed:
[DEPLOY-310] Update the DBP deploy to use the common components we moved from the Activiti org

Update the helm repo add command.  Remove -test  from the repo name.

jottley committed:
[DEPLOY-310] Update the DBP deploy to use the common components we moved from the Activiti org

Fix name of the helm repo add to be alfresco-incubator
